node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodesV2ApiHandler.java

```diff
@@ -116,6 +116,7 @@ private HttpResponse handleGET(HttpRequest request) {
         if (pathS.startsWith("/nodes/v2/state/")) return new NodesResponse(ResponseType.nodesInStateList, request, orchestrator, nodeRepository);
         if (pathS.startsWith("/nodes/v2/acl/")) return new NodeAclResponse(request, nodeRepository);
         if (pathS.equals(    "/nodes/v2/command/")) return new ResourceResponse(request.getUri(), "restart", "reboot");
+        if (pathS.equals(    "/nodes/v2/locks")) return new LocksResponse();
```

Comment:
Fine for now, but I think we should extract this into a top-level `/debug/` API if we intend to keep it.

Reply:
+1